install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -4,6 +4,11 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+INSTALL_COCO_CONTAINERD=${INSTALL_COCO_CONTAINERD:-false}
+INSTALL_OFFICIAL_CONTAINERD=${INSTALL_OFFICIAL_CONTAINERD:-true}
```

Comment:
Should the default be set to false given the default is to rely on installed containerd ?

Reply:
I've set the default there to false, that was my bad.